src/chart/barChartMultiSeries.js

```diff
@@ -86,7 +82,7 @@ export default function() {
 			.data(layers)
 			.enter()
 			.append("group")
-			.attr("class", function(d) { return d; });
+			.attr("class", d => d);
```

Comment:
I have put my head in sand when it comes to ES6 arrow functions.. I think I need to take the leap and embrace them! You have spurred me on!

Reply:
Just keep in mind to use a normal function if you need `this`.